pkgs/tools/misc/docui/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  name = "docui-${version}";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "skanehira";
+    repo = "docui";
+    rev = version;
+    sha256 = "1kbap36hccwlj273is98cvgf5z5cl2c3s6p46nh6bnykz3zqzs71";
+  };
+
+  modSha256 = "0qxlrjwngqr1yb29jd5wkd3b9c5igdii9s6cqy3grdz90flfk1av";
+
+  meta = with stdenv.lib; {
+    description = "TUI Client for Docker";
+    homepage = https://github.com/skanehira/docui;
+    license = licenses.mit;
+    maintainers = [ maintainers.aethelz ];
+    platforms = platforms.linux;
+  };
+}
```

Comment:
Can you please re-indent this file. Thanks.

Reply:
Re-indent? I checked a few packages, and tried to follow the same conventions. Should I use tabs instead of spaces?

Reply:
Also, is it okay to add you as a package maintainer for the time being?

Reply:
> Re-indent? I checked a few packages, and tried to follow the same conventions. Should I use tabs instead of spaces?

:man_facepalming: I think GitHub changed something, but I can clearly see a few spaces before the first line `{ stdenv, buildGoModule, fetchFromGitHub }:` but I cannot see a space when I try to select the code. Never mind then.

http://i.imgur.com/C5CkA0I.png




> Also, is it okay to add you as a package maintainer for the time being?

I'm the default maintainer for all Go packages ([no need to add me explicitly](https://github.com/NixOS/nixpkgs/blob/0a1ed299f443f31bd5d0b849540002a5f15104c1/pkgs/development/go-modules/generic/default.nix#L198)), but some help would be nice if you can commit to that.

Reply:
Okay, thank you!

I'm a bit confused by contributions guidelines though. In nixos manual it says that maintainers field is optional, CONTRIBUTING.md.on the other hand says that:

**meta.maintainers must be set.**

So can I just leave out this field or not for now?

Reply:
`meta.maintainers` is a required field. Although, `buildGoModule` itself sets it to me + the value you set yourself. So even if you leave it off, it'll get set by `buildGoModule` itself.